pluspy.py

```diff
@@ -774,6 +774,30 @@ def parseError(a, r):
         shortest = r
     return (False, a, r)
 
+
+
+# Handy routine for rules that simply call other rules
+# It parses s using the given rule and returns a node (name, (t, a), r)
+# where name is the type of the AST node and (t, a) the result of
+# parsing given the rule.
+#
+# If t is a "Concat" rule (sequence of other rules), then you can
+# select a subset using the select argument
+#
+# Otherwise, if select is not None, the result of the rule is
+# directly returned without adding a new AST node
+def match(name, s, rule, select=None):
+    (t, a, r) = rule.parse(s)
+    if not t:
+        return parseError([name] + a, r)
+    if isinstance(select, list) and t == "Concat":
+        if len(select) == 1:
+            return (name, a[select[0]], r)
+        return (name, [ a[i] for i in select ], r)
+    if select != None:
```

Comment:
[PEP 8 says](https://www.python.org/dev/peps/pep-0008/#programming-recommendations):

> Comparisons to singletons like `None` should always be done with `is` or `is not`, never the equality operators.

([Except for strings](https://stackoverflow.com/a/1504740/1959808), which are compared using `==`, not `is`.)